src/transformers/models/udop/processing_udop.py

```diff
@@ -93,6 +116,19 @@ def __call__(
         Please refer to the docstring of the above two methods for more information.
         """
         # verify input
+        output_kwargs = self._merge_kwargs(
+            UdopProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
+            **kwargs,
+            **self.prepare_and_validate_optional_call_args(*args),
+        )
+
+        boxes = output_kwargs["text_kwargs"].pop("boxes", None)
+        word_labels = output_kwargs["text_kwargs"].pop("word_labels", None)
+        text_pair = output_kwargs["text_kwargs"].pop("text_pair", None)
+        return_overflowing_tokens = output_kwargs["text_kwargs"].get("return_overflowing_tokens", False)
+        return_offsets_mapping = output_kwargs["text_kwargs"].get("return_offsets_mapping", False)
```

Comment:
ugh, lots of additional args here - but no way around it I suppose. Maybe we could group a bit by default values using iterators, for style and loc, but minor nit

Reply:
Yes it's not very nice... But grouping by default values and doing list comps is a bit hard to read here I feel like

Reply:
Yup, agreed